api-docs/influxdb3/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml

```
@@ -148,6 +283,38 @@ paths:
           text/plain:
             schema:
               type: string
+          application/json:
+            schema:
+              type: object
+              properties:
+                db:
+                  type: string
+                  description: Bucket to query.
+                q:
```

```suggestion
                rp:
                  description: |
                    The retention policy to query data from. For more information, see [InfluxQL DBRP naming convention](/influxdb3/cloud-dedicated/admin/databases/create/#influxql-dbrp-naming-convention).
                  type: string
                q:
```

I assume we missed retention policy as well.